catalog/app/containers/Bucket/PackageDialog/PackageDialog.js

```diff
@@ -104,16 +124,18 @@ function mkMetaValidator(schema) {
   return function validateMeta(value) {
     const noError = undefined
 
+    const jsonObjectErr = validators.jsonObject(value.text)
+    if (jsonObjectErr) {
+      return value.mode === 'json'
+        ? jsonObjectErr
+        : [{ message: 'Metadata must be a valid JSON object' }]
```

Comment:
There is `ERROR_MESSAGES` at the top of the module. I think we can use it to store this string

Reply:
i thought that constant is for reusable messages related to server responses mostly